support/elasticsearch.erl

```diff
@@ -67,6 +69,18 @@ delete_doc(Id, Index, _Context) ->
     ),
     handle_response(Response).
 
+delete_doc(Id, Type, Index, _Context) ->
+    Response = erlastic_search:delete_doc(
+        Index, Type, Id
+    ),
+    handle_response(Response).
+
+get_doc(Id, Index, _Context) ->
+    Response = erlastic_search:get_doc(
+        Index, "_all", Id
```

Comment:
While this is a nice shortcut, it can create trouble when two documents in different types that have the same id, e.g. `/mickey/1`, `/mouse/2`. Moreover, this extra layer isn’t really necessary as you can just call `erlastic_search:get_doc/3`.